### ISSUE TITLE
installation fails due to db error "primary index name ... too long"

### ISSUE THREAD

**Issue by guzzisti** [nextcloud/social#506]
**Describe the bug**
App can not be installed on NC 16.0.0 as a error during db-migration occurs.

**To Reproduce**
1. Go to Social on the Apps-Page of your nextcloud instance
2. Click on 'Download and enable'
3. Error message *Primary index name on "prefix_social_server_follows" is too long.*
4. App is not installed.

**Expected behavior**
App is installed correctly with needed db

<details>
<summary>Server details</summary>

**Social app version:**
0.14.0

**Operating system**:
Ubuntu

**Web server:**
Apache

**Database:**
MySQL 5.7.25

**PHP version:**
7.3.3

**Nextcloud version:** (see Nextcloud admin page)
16.0.0

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
[settings] Error: InvalidArgumentException: Primary index name  on "prefix_social_server_follows" is too long. at <<closure>>

 0. /www/htdocs/nextcloud/lib/private/DB/MigrationService.php line 481
    OC\DB\MigrationService->ensureOracleIdentifierLengthLimit(Doctrine\DBAL\Schema\Schema {}, Doctrine\DBAL\Schema\Schema {}, 6)
 1. /www/htdocs/nextcloud/lib/private/DB/MigrationService.php line 411
    OC\DB\MigrationService->executeStep("0001Date20181204203147", false)
 2. /www/htdocs/nextcloud/lib/private/Installer.php line 141
    OC\DB\MigrationService->migrate()
 3. /www/htdocs/nextcloud/settings/Controller/AppSettingsController.php line 447
    OC\Installer->installApp("social")
 4. /www/htdocs/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 166
    OC\Settings\Controller\AppSettingsController->enableApps(["social"], [])
 5. /www/htdocs/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 99
    OC\AppFramework\Http\Dispatcher->executeController(OC\Settings\Cont ... {}, "enableApps")
 6. /www/htdocs/nextcloud/lib/private/AppFramework/App.php line 126
    OC\AppFrameworkationService.php line 481
\Http\Dispatcher->dispatch(OC\Settings\Cont ... {}, "enableApps")
 7. /www/htdocs/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
    OC\AppFramework\App::main("OC\\Settings\\C ... r", "enableApps", OC\AppFramework\ ... {}, {_route: "settin ... "})
 8. <<closure>>
    OC\AppFramework\Routing\RouteActionHandler->__invoke({_route: "settin ... "})
 9. /www/htdocs/nextcloud/lib/private/Route/Router.php line 297
    undefinedundefinedcall_user_func(OC\AppFramework\ ... {}, {_route: "settin ... "})
10. /www/htdocs/nextcloud/lib/base.php line 975
    OC\Route\Router->match("/settings/apps/enable")
11. /www/htdocs/nextcloud/index.php line 42
    OC::handleRequest()

POST /index.php/settings/apps/enable
```
</details>

**Comment by daita** [comment]
fixed in alpha2

**Comment by guzzisti** [comment]
Issue has been fixed for me after upgrading to NC 16.0.1. I guess it was related to https://github.com/nextcloud/server/pull/15312

**Comment by rubo77** [comment]
In which commit was this fixed? Maybe it is a solution to the new problem in Nextcloud 18 too #850
